### PR TITLE
Support LHM methods with columns as symbols

### PR DESCRIPTION
**Pull request by sauloperez**

### CODE REVIEW

**Comment by Morantron** [comment]
:eyes:

File: lib/percona_migrator/lhm/fake/adapter.rb [hunk 1]
```diff
@@ -23,14 +23,18 @@ def initialize(migration, table_name)
...
+          else
+            column = column(column_name, definition)
+
+            migration.add_column(
+              table_name,
+              column_name,
+              column.type,
+              column.to_hash
+            )
+          end
```
Comment (enricostano) [inline lib/percona_migrator/lhm/fake/adapter.rb]: maybe we can have something like
**ALERT: pseudo code ahead**

```
def add_column(column_name, definition)
  attributes = if definition.is_a?(Symbol)
                      # 3 attributes...
                    else
                      # 4 attributes
                    end

  migration.add_column(*attributes)
end
```

Comment (sauloperez) [inline lib/percona_migrator/lhm/fake/adapter.rb]: Yes, the different number of attributes involves something like that. However, it'd be simpler to pass an empty options hash (last argument) if possible. In any case, I'll try to dry that if.

**Comment by sauloperez** [comment]
I finally followed your wise advice @enricostano :ok_hand:

**Comment by enricostano** [comment]
:heart_eyes: spreading love like attributes :wink:

**Pull request merged**